File: src/RummyAi.Domain/Features/PlayerDto/Model/PlayerCards.cs
using RummyAi.Domain.Features.DeckDto.Models;
using RummyAi.Domain.Features.GameDto;

namespace RummyAi.Domain.Features.PlayerDto.Model;

public class PlayerCards
{
    public PlayerId PlayerId { get; init; }
    public Stack Stack { get; set; }

    public PlayerCards(PlayerId playerId, Stack cards)
    {
        PlayerId = playerId;
        Stack = cards;
    }
}

File: src/RummyAi.WebClient/domain/models/PlayerCards.ts
import type { PlayerId } from "./PlayerId";
import type { Stack } from "./Stack";

export class PlayerCards {
    playerId: PlayerId
    stack: Stack

    public constructor(
        playerId: PlayerId,
        stack: Stack) {
        this.playerId = playerId;
        this.stack = stack;
    }
}